packages/util_linux.rb

```diff
@@ -38,78 +38,14 @@ class Util_linux < Meson
-  meson_options "#{year2038} #{disabled_builds}"
-
-  def self.patch
-    # Fix undefined reference to `pthread_atfork' build error
-    # introduced by https://github.com/util-linux/util-linux/pull/3017
-    # and mentioned in https://github.com/util-linux/util-linux/issues/3131 .
-    system "sed -i \"1i thread_dep = dependency('threads')\" libuuid/meson.build"
-    system "sed -i \"s/dependencies : \\\[socket_libs,/dependencies : \\\[socket_libs, thread_dep,/\" libuuid/meson.build"
-  end
-
-  def self.install
-    system "DESTDIR=#{CREW_DEST_DIR} #{CREW_NINJA} -C builddir install"
-    return if ARCH == 'i686'
-
-    # Imagemagick wants a libuuid libtool file.
-    @libname = 'libuuid'
-    @libnames = Dir["#{CREW_DEST_LIB_PREFIX}/#{@libname}.so*"]
-    @libnames = Dir["#{CREW_DEST_LIB_PREFIX}/#{@libname}-*.so*"] if @libnames.empty?
-    @libnames.each do |s|
-      s.gsub!("#{CREW_DEST_LIB_PREFIX}/", '')
-    end
-    @dlname = @libnames.grep(/.so./).first
-    @dlname = @libnames.grep(/.so/).first if @dlname.nil?
-    @libname = @dlname.gsub(/.so.\d+/, '')
-    @longest_libname = @libnames.max_by(&:length)
-    @libvars = @longest_libname.rpartition('.so.')[2].split('.')
-    @libtool_file = <<~LIBTOOLEOF
-      # #{@libname}.la - a libtool library file
-      # Generated by libtool (GNU libtool) (Created by Chromebrew)
-      #
-      # Please DO NOT delete this file!
-      # It is necessary for linking the library.
-
-      # The name that we can dlopen(3).
-      dlname='#{@dlname}'
-
-      # Names of this library.
-      library_names='#{@libnames.reverse.join(' ')}'
-
-      # The name of the static archive.
-      old_library='#{@libname}.a'
-
-      # Linker flags that cannot go in dependency_libs.
-      inherited_linker_flags=''
-
-      # Libraries that this one depends upon.
-      dependency_libs=''
-
-      # Names of additional weak libraries provided by this library
-      weak_library_names=''
-
-      # Version information for #{name}.
-      current=#{@libvars[1]}
-      age=#{@libvars[1]}
-      revision=#{@libvars[2]}
-
-      # Is this an already installed library?
-      installed=yes
-
-      # Should we warn about portability when linking against -modules?
-      shouldnotlink=no
-
-      # Files to dlopen/dlpreopen
-      dlopen=''
-      dlpreopen=''
-
-      # Directory that this library needs to be installed in:
-      libdir='#{CREW_LIB_PREFIX}'
-    LIBTOOLEOF
-    File.write("#{CREW_DEST_LIB_PREFIX}/#{@libname}.la", @libtool_file)
-  end
+  year2038 = '--disable-year2038'
+  i686_disabled_builds = '--disable-blkzone --disable-lsfd'
+  autotools_configure_options "#{year2038 unless ARCH == 'x86_64'} \
+                              --disable-kill \
+                              #{i686_disabled_builds if ARCH == 'i686'} \
+                              --disable-makeinstall-chown \
+                              --disable-makeinstall-setuid \
+                              --disable-makeinstall-tty-setgid \
```

Comment:
Are these disabled available from another package?

Reply:
The disabled settings are as per before, aside from the makeinstall disables that would otherwise require sudo.